lib/locales/es.json

```diff
@@ -0,0 +1,434 @@
+        "failedToSwitchAccount": "Error al cambiar cuenta",
+        "accountSignedOutSwitched": "Cuenta desconectada. Cambiado a otra cuenta disponible.",
+        "signedOutSuccessfully": "Desconectado exitosamente.",
+        "failedToCheckAccountsAfterLogout": "Error al verificar cuentas después del cierre de sesión",
+        "signOutTitle": "¿Cerrar sesión?",
+        "signOutDescription": "Estás seguro de querer cerrar sesión? Si no has hecho una copia de seguridad de tus llaves, no podrás recuperarlas.",
+        "privacyAndSecurity": "Privacidad y Seguridad",
+        "deleteAllData": "Borrar todos los datos",
+        "deleteAllDataTitle": "¿Estás seguro que quieres borrar todo?",
+        "deleteAllDataDescription": "Esto eliminará todos los datos de grupos, mensajes y cerrará sesión de todas tus cuentas. Esto no borrará tus llaves de nostr ni otros eventos que hayas publicado. <br><br> ¿Estás seguro de querer borrar todos los datos de White Noise? Esto no se puede deshacer.",
+        "publishAKeyPackage": "Publicar un paquete de llaves",
+        "publishKeyPackage": "Publicar paquete de llaves",
+        "publishKeyPackageDescription": "¿Estás seguro de querer publicar un nuevo evento de paquete de llaves?",
+        "deleteAllKeyPackages": "Eliminar todos los paquetes de llaves",
+        "deleteAllKeyPacagesDescription": "¿Estás seguro de querer eliminar todos los paquetes de llaves?",
+        "testNotifications": "Probar notificaciones",
+        "notificationSuccess": "Prueba de notificación exitosa",
+        "toastSuccess": "Mensaje de éxito",
+        "testToastSuccess": "Probar mensaje de éxito",
+        "toastError": "Mensaje de error",
+        "testToastError": "Probar mensaje de error",
+        "qrCode": "Tu Llave Pública",
+        "developerSettings": "Configuración de desarrollador",
+        "keyPackageManagement": "Gestión de paquetes de llaves",
+        "publishNewKeyPackage": "Publicar nuevo paquete de llaves",
+        "inspectRelayKeyPackages": "Inspeccionar paquetes de llaves del relé",
+        "deleteAllKeyPackagesFromRelays": "Eliminar todos los paquetes de llaves de los relés",
+        "deleteAllKeyPackagesTitle": "Eliminar todos los paquetes de llaves",
+        "deleteAllKeyPackagesDescription": "Esto eliminará todos los paquetes de llaves de la cuenta activa. Otros usuarios no podrán invitarte a nuevas conversaciones encriptadas hasta que generes nuevos paquetes de llaves. Esta acción no se puede deshacer.",
+        "deleteKeyPackageTitle": "Eliminar paquete de llaves",
+        "deleteKeyPackageDescription": "Esto eliminará el paquete de llaves #{number}. Otros usuarios no podrán usar este paquete de llaves para invitarte a nuevas conversaciones encriptadas. Esta acción no se puede deshacer.",
+        "keyPackagesCount": "Paquetes de llaves ({count})",
+        "noKeyPackagesFound": "No se encontraron paquetes de llaves",
+        "keyPackageNumber": "Paquete de llaves #{number}",
```

Comment:
`Paquete de llaves N°{number}` (I feel in spanish the use of # is less common than in english)